CMSgov/design-system
# [LHDM-2201] Header/Footer Custom classes/IDs

## Summary

- Our team has a specific Smartling-related requirement to be able to assign class names and IDs to the logo, language switch link, and log in link in the Header and the logo in the Footer.
- https://jira.cms.gov/browse/LHDM-2201 

**Update:** After chatting, this has been simplified to only include the custom classes.

## How to test

Use the story controls to apply the various new props. (Note that you'll have to use both the logged in and logged out stories to see all the props.)

## Checklist




### If this is a change to design:


### If this is a change to code:


### If this is a change to documentation:
